mosip-acceptance-tests/ivv-orchestrator/entrypoint.sh

```diff
@@ -15,4 +15,4 @@ fi
 
 ## Run DSL Orchestrator
 java --version
-java -jar -Denv.user="$USER" -Denv.endpoint="$ENDPOINT" -Denv.testLevel="$TESTLEVEL" ivv-orchestrator-*-jar-with-dependencies.jar
+java -jar -Denv.user="$USER" -Denv.endpoint="$ENDPOINT" -Denv.testLevel="$TESTLEVEL" dslrig-ivv-orchestrator*.jar-with-dependencies.jar
```

Comment:
update `dslrig-ivv-orchestrator*.jar-with-dependencies.jar` 
to `dslrig-ivv-orchestrator-*-jar-with-dependencies.jar`